scripts/assemble_figure.py

```
@@ -0,0 +1,376 @@
+                if chunk.strip().startswith("font-family"):
+                    parts.append(f"font-family:{font_family}")
+                else:
+                    parts.append(chunk)
+            elem.set("style", ";".join(parts))
+            count += 1
+
+    tree.write(str(svg_path), xml_declaration=True, encoding="utf-8")
+    print(f"[font] normalized {count} elements -> {font_family}")
+
+
+def export_pdf(svg_path: Path, pdf_path: Path, outline_text: bool = False) -> None:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)